src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/BodySubstitution.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Internal.IL;
+using Internal.IL.Stubs;
+using Internal.TypeSystem;
+
+namespace ILCompiler
+{
+    internal class BodySubstitution
+    {
+        private object _value;
+
+        private readonly static object Throw = new object();
+
+        public readonly static BodySubstitution ThrowingBody = new BodySubstitution(Throw);
+        public readonly static BodySubstitution EmptyBody = new BodySubstitution(null);
+
+        public object Value
+        {
+            get
+            {
+                Debug.Assert(_value != Throw);
+                return _value;
+            }
+        }
+
+        private BodySubstitution(object value) => _value = value;
+
+        public static BodySubstitution Create(object value) => new BodySubstitution(value);
+        public MethodIL EmitIL(MethodDesc method)
+        {
+            ILEmitter emit = new ILEmitter();
+            ILCodeStream codestream = emit.NewCodeStream();
+
+            if (_value == Throw)
+            {
+                codestream.EmitCallThrowHelper(emit, method.Context.GetHelperEntryPoint("ThrowHelpers", "ThrowFeatureBodyRemoved"));
```

Comment:
```suggestion
                codestream.EmitCallThrowHelper(emit, method.Context.GetHelperEntryPoint(nameof(ThrowHelpers), nameof(ThrowHelpers.ThrowFeatureBodyRemoved)));
```

If ThrowHelpers are accessible here, please use `nameof`

Reply:
It's not accessible.